### ISSUE TITLE
Compliance.urlencode still takes a separator symbol

### ISSUE THREAD

**Issue by jspaaks** [fair-software/howfairis#300]
But should it? https://github.com/fair-software/howfairis/blob/738f2f9a0c5fc4409199af9bb189bc86b8dfaa47/howfairis/compliance.py#L108

**Comment by sverhoeven** [comment]
Agree, should be same as in Readme class. https://github.com/fair-software/howfairis/blob/0b575cf66acf56edb9799faf0ba028566123c835/howfairis/readme.py#L27
